src/jit/rangecheck.h

```diff
@@ -191,7 +191,7 @@ struct Limit
                 return "Dependent";
 
             case keBinOpArray:
-                sprintf_s(buf, size, "VN%04X + %d", vn, cns);
+                sprintf_s(buf, size, FMT_VN " + %d", vn, cns);
```

Comment:
Here and in a few other places width (%04 here) is removed from format. Do you think it won't matter for dump readability?

Reply:
I will change these as well.  I prefer that everyone prints them the same way that was my searches in the dump with find all occurances of $83 and not miss this one that print out currently as $0083